### PR TITLE
Hightlight results

### PR DESCRIPTION
**Pull request by abhishek77in**

### CODE REVIEW

File: app/assets/javascripts/lab_reports.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+$(function(){
+
+  $(".numeric-test-result").on('change paste keyup', function(){
+    var lowerValue = $(this).data('lower-value');
+    console.log(lowerValue);
+    var upperValue = $(this).data('upper-value');
+    console.log(upperValue);
+    var result = $(this).val();
+    console.log(result);
+    $(this).closest('tr').css("font-weight", "bold");
```
Comment (abhishek77in) [inline app/assets/javascripts/lab_reports.js]: Fixed the highlighting. Please take care of adding conditional highlighting and reset.

Comment (abhishek77in) [reply app/assets/javascripts/lab_reports.js]: @bajajerk .

**Comment by bajajerk** [comment]
Please review @abhishek77in

**Comment by abhishek77in** [comment]
@bajajerk LGTM

**Pull request merged**